anoncreds_test.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+from controller.controller import Controller
```

Comment:
I don't see this `controller` package, does it need to be added?
This demo might be better placed under `examples/`

Reply:
Sorry for the confusion, this is part of a quick test script that I intend to remove before merging to main, replacing it with some proper testing. This script is run by a container built from `docker/Dockerfile.anoncreds-test`, which does a quick `pip install` of the needed library. This gets run by the `tests` service in `docker-compose.yml` (which will also be removed before merging). We just found having a quick high-level test helpful while things are in flight.